dist/windows/installer.nsi

```diff
@@ -22,24 +22,20 @@ Section $(inst_qbt_req) ;"qBittorrent (required)"
 
   SectionIn RO
 
-  ; Set output path to the installation directory.
-  SetOutPath $INSTDIR
-
   ;Create 'translations' directory
   CreateDirectory $INSTDIR\translations
 
-  ; Put file there
+  ; Set output path to the installation directory.
+  SetOutPath $INSTDIR
+  ; Put files there
   File "qbittorrent.exe"
   File "qbittorrent.pdb"
   File "qt.conf"
-  File /r "qtbase_*.qm"  ; omit translations folder path to preserve folder structure
-  File /oname=translations\qt_fa.qm "translations\qt_fa.qm"
-  File /oname=translations\qt_gl.qm "translations\qt_gl.qm"
-  File /oname=translations\qt_lt.qm "translations\qt_lt.qm"
-  File /oname=translations\qt_pt.qm "translations\qt_pt.qm"
-  File /oname=translations\qt_sl.qm "translations\qt_sl.qm"
-  File /oname=translations\qt_sv.qm "translations\qt_sv.qm"
-  File /oname=translations\qt_zh_CN.qm "translations\qt_zh_CN.qm"
+
+  ; Set output path to the installation\translations directory.
+  SetOutPath "$INSTDIR\translations"
+  ; Put files there
+  File /r "translations\qt*.qm"
```

Comment:
Now line 47 in Readme.txt should be removed too. IMO it would be even better if you can provide a small python script (or in any language) to automate the operations described in that section (qt translations section). Something like a commandline program that user provides path to qt translations file and it will copy the needed files to a specified folder.

Reply:
The python script is already provided by a previous refactor in the summer.